From 117f29e7c5880e178b29bd8fedc37c745192b975 Mon Sep 17 00:00:00 2001
From: Swee Yee Fonn <swee.yee.fonn@intel.com>
Date: Wed, 4 Nov 2020 18:57:34 +0800
Subject: [PATCH] Update sepolicy for keybox_provisioning

Update sepolicy to allow keybox provisioning over
vsocket.

Tracked-On: OAM-95582
Change-Id: I733d33f2eb7eb8f2014e5d6eb1050a3c9f174981
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>
---
 trusty/enabled/tee.te | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/trusty/enabled/tee.te b/trusty/enabled/tee.te
index 0091ad2..81fafbf 100644
--- a/trusty/enabled/tee.te
+++ b/trusty/enabled/tee.te
@@ -11,3 +11,8 @@ allow tee rpmb_virt_device:chr_file { open read write };
 allow tee self:capability sys_rawio;
 allow tee block_device:dir search;
 allow tee tee_device:blk_file rw_file_perms;
+
+allow tee self:vsock_socket create;
+allow tee self:vsock_socket connect;
+allow tee self:vsock_socket read;
+allow tee self:capability net_admin;
--
2.17.1

